CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+## Install rust
+
+You can follow instructions at ['Install Rust' page from the official rust website](https://www.rust-lang.org/tools/install)
+
+## Add wasm target to rust
+
+```bash
+rustup target add wasm32-wasi
+```
+
+## Running tests
+```bash
+# run all test suite
+cargo test
+
+# run individual test
+cargo test js_choices_may_contain_expressions
+
+# you may specify only prefix of test name to target more cases
+cargo test jsx_
+```
+
+## Building for production
+
+```bash
+# (alias for `cargo build --target wasm32-wasi`)
+cargo build-wasi --release
+```
+Then wasm binary would be on the path: `./target/wasm32-wasi/release/lingui_macro_plugin.wasm`
+
+You can check it in your own project or in the `examples/nextjs-13` example in this repo by specifying full path to the WASM binary:
+
+```ts
```

Comment:
maybe add a section for plain swc integration ? here is the config in `swcrc` we are using:

```
module.exports = {
  $schema: 'http://json.schemastore.org/swcrc',
  jsc: {
    experimental: {
      plugins: [
        // ... other plugins
        ['@lingui/swc-plugin', {}],
      ],
    },
  },
  minify: false,
}
```